kernel/unwind.c

```diff
@@ -505,7 +505,7 @@ static int find_fde(uint64_t rip, eh_cie* cie_out, eh_fde* fde_out) {
     memset(&fde, 0, sizeof(eh_fde));
     fde.cie = &current_cie;
     if (read_fde(&subhead, &fde)) { return -1; }
-    if (rip >= fde.pc_begin && rip - fde.pc_begin < fde.pc_len) {
+    if (rip >= fde.pc_begin && rip - fde.pc_begin <= fde.pc_len) {
```

Comment:
Do you want > for the first condition?  Right now, I think you'll get two ranges overlapping at one address